api/wasm/cpp/BUILD

```diff
@@ -0,0 +1,20 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+ 
```

Comment:
Nit: extra empty line, perhaps run `buildifer` on this file?

Reply:
done.